sites/test-site/src/data/pages/cx/header/header$image.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
i think we'd rather have the logo itself with a node and nodekey, so this path becomes header$logo$image.json